picamera2/picamera2.py

```diff
@@ -411,21 +505,46 @@ def align_stream(self, stream_config: dict) -> None:
         stream_config["size"] = (size[0] - size[0] % align, size[1] - size[1] % 2)
 
     def is_YUV(self, fmt) -> bool:
-        return fmt in ("NV21", "NV12", "YUV420", "YVU420", "YVYU", "YUYV", "UYVY", "VYUY")
+        return fmt in (
+            "NV21",
+            "NV12",
+            "YUV420",
+            "YVU420",
+            "YVYU",
+            "YUYV",
+            "UYVY",
+            "VYUY",
+        )
 
```

Comment:
I think the original single line version looks better.

Reply:
Concur.